lib/validators/liveValidator.ts

```diff
@@ -158,6 +165,12 @@ export class LiveValidator {
     this.options = ops as LiveValidatorOptions
   }
 
+  private loggingCallback(message: string, level: number = winston.config.npm.levels.info): void {
```

Comment:
Could you use string as level directly?  I don't think oav need to depend on winston

Reply:
Updated.